### PR TITLE
Minor documentation updates

### PR DESCRIPTION
**Pull request by abhro**
- Array/Vector/Matrix output showing
- Syntax highlighting for fenced code block examples

### CODE REVIEW

**Review by ViralBShah — APPROVED** [review]

File: base/docs/basedocs.jl [hunk 1]
```diff
@@ -3014,7 +3014,7 @@ an alias for `UndefInitializer()`.
 # Examples
 ```julia-repl
 julia> Array{Float64, 1}(UndefInitializer(), 3)
-3-element Array{Float64, 1}:
+3-element Vector{Float64, 1}:
```
Comment (LilithHafner) [inline base/docs/basedocs.jl:3017]:
```suggestion
3-element Vector{Float64}:
```

File: doc/src/manual/distributed-computing.md [hunk 2]
```diff
@@ -1371,7 +1371,7 @@ julia> all(C .≈ 4*π)
 true
 
 julia> typeof(C)
-Array{Float64,1}
+Vector{Float64,1} (alias for Array{Float64, 1})
```
Comment (LilithHafner) [inline doc/src/manual/distributed-computing.md:1374]:
```suggestion
Vector{Float64} (alias for Array{Float64, 1})
```

**Review by LilithHafner — COMMENTED** [review]
Thank you! This is a lovely docs improvement.

File: base/multidimensional.jl [hunk 3]
```diff
@@ -1322,7 +1322,7 @@ julia> src = reshape(Vector(1:16), (4,4))
 julia> dest = OffsetArray{Int}(undef, (0:3,2:5))
```
Comment (inkydragon) [inline base/multidimensional.jl:1322]:
```suggestion
julia> dest = OffsetArray{Int}(undef, (0:3,2:5));
```

**Review by LilithHafner — APPROVED** [review]

**Pull request merged**